pkg/controllers/node/termination/metrics.go

```diff
@@ -24,9 +24,17 @@ import (
 )
 
 func init() {
-	crmetrics.Registry.MustRegister(TerminationDurationSeconds)
+	crmetrics.Registry.MustRegister(
+		TerminationDurationSeconds,
+		NodeLifetimeDurationSeconds)
 }
 
+const (
+	minuteInSeconds = 60
```

Comment:
Can you just represent these in the buckets as `time.Minute.Seconds()`, `time.Hour.Seconds()`, etc.?

Reply:
`time.Minute` and `time.Hour` are represented in milliseconds. So I didn't use them. But now I see there is a `Seconds` function for them. Ok, will make this change.